server/src/main/java/org/apache/druid/server/coordinator/CoordinatorDynamicConfig.java

```diff
@@ -495,6 +533,8 @@ private static class Defaults
     static final int MAX_NON_PRIMARY_REPLICANTS_TO_LOAD = Integer.MAX_VALUE;
     static final boolean USE_ROUND_ROBIN_ASSIGNMENT = true;
     static final boolean SMART_SEGMENT_LOADING = true;
+    static final double KILL_TASK_SLOT_RATIO = 1.0;
+    static final int MAX_KILL_TASK_SLOTS = Integer.MAX_VALUE;
```

Comment:
A comment here that these defaults are to preserve the behavior before Druid 0.28 and a future version may want to consider better defaults so that kill tasks can not eat up all the capacity in the cluster

Reply:
added, thanks